content/en/continuous_integration/pipelines/teamcity.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Set up Tracing on a TeamCity Pipeline
+kind: documentation
+aliases:
+  - /continuous_integration/setup_pipelines/teamcity
+further_reading:
+  - link: "/continuous_integration/pipelines"
+    tag: "Documentation"
+    text: "Explore Pipeline Execution Results and Performance"
+  - link: "/continuous_integration/troubleshooting/"
+    tag: "Documentation"
+    text: "Troubleshooting CI"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Compatibility
+
+- **Supported TeamCity versions**: 2021.2 or higher
+
+- **Partial retries**: View partially retried pipeline executions
+
+- **Queue time**: View amount of time pipeline jobs sit in the queue before processing
+
+- **Pipeline failure reasons**: Identify pipeline failure reasons from error messages
+
+## Configure the Datadog integration
+
+The integration between [TeamCity][1] and Datadog CI Visibility is provided through a TeamCity plugin.
+To activate the integration:
+1. Download the [Datadog CI Integration plugin][5] on the TeamCity server by going to
+Administration -> Plugins -> Browse Plugin Repository.
+2. For the integration to work properly, a [TeamCity composite build][6] must be added as the last build
```

Comment:
Maybe there's a screenshot you can provide with exactly what people should input there?

Reply:
For the composite build? Unfortunately the dependencies they need to set really depend on how they structured their build chain :/ The only screenshot I can provide is how to create a composite build, but it's basically a "check" when creating a build, to define that it will be composite

Reply:
understood, I think this makes sense to add it anyway and maybe show what it looks like (a random pipeline with a composition build in the end). Sorry if it does not make sense, feel free to ignore

Reply:
I've added a sample pipeline as an image, looks fine to me. WDYT?